### ISSUE TITLE
R2modman support for staging profile

### ISSUE THREAD

**Issue by Lordfirespeed** [Cryptoc1/lc-plugin-sdk#1]
r2modman profiles are kept here:
```
$(APPDATA)/r2modmanPlus-local/LethalCompany/profiles/[profile name]
```

Edit: I know you can set `PluginStagingDir` manually. It would be nice to not have to.

**Comment by Cryptoc1** [comment]
@Lordfirespeed Support for r2modman is slated for release in version `v1.1.0`. I've pushed a release candidate, [`v1.1.0-rc.3`](https://www.nuget.org/packages/LethalCompany.Plugin.Sdk/1.1.0-rc.3), for testing if you'd like to try it out until the official release (note however, as this is a pre-release, there may be a few bugs to work out).

Full details of the changes I've made can be found in the summary of #2.

> _This issue will automatically be closed when `v1.1.0` is officially released._

## _TL;DR_

- Update to `v1.1.0-rc.3`:
```xml
<Project Sdk="LethalCompany.Plugin.Sdk/1.1.0-rc.3">
  ...
</Project>
```

- Set the `PluginManager` property:
```xml
<PropertyGroup>
  ...

  <PluginManager>r2modman</PluginManager>
</PropertyGroup>
```

> _I recommend setting this property in the `.csproj.user` file, adjacent to `<PluginStagingProfile />`_

-  The Sdk now auto-references assemblies for `ThunderDependency` items. If you're using `ThunderDependency` items, you no longer need to specify manual `Reference`s.

**Comment by Lordfirespeed** [comment]
Awesome! 

Closed by #2
